features/FEATURE_BLE/targets/TARGET_CORDIO/CordioPalAttClient.h

```diff
@@ -216,7 +216,7 @@ class CordioAttClient : public ::ble::pal::AttClient {
         AttcSignedWriteCmd(
             connection_handle,
             attribute_handle,
-            /* sign counter from flash or AttsGetSignCounter() ? */ 0,
+            /*TODO: get sign counter from cordio sm */0,
```

Comment:
We need to get that number from the SM.

Reply:
the counter is set during init when the CSRK is set and should be handled by the implementation and not supplied per write, I am working on a sane solution for this right now